src/instructlab/chat/chat.py

```diff
@@ -482,7 +482,10 @@ def chat_cli(
         http_client=httpx.Client(cert=cert, verify=verify),
     )
     # ensure the model specified exists on the server. with backends like vllm, this is crucial.
-    model_list = client.models.list().data
+    try:
+        model_list = client.models.list().data
+    except (openai.APIConnectionError, httpx.ConnectError) as exc:
```

Comment:
generally, we catch only `openai.APIConnectionError`, but i left `httpx.ConnectError` as well.

Reply:
Sure, done.